pkg/hs/urbit-king/lib/Urbit/Vere/Serf/IPC.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE StrictData #-}
```

Comment:
Since IPC defines not only the ipc types (analogous to ev/ef types) but also complex dataflows, I feel less confident about this than I do about ev/ef. What if billy, in the course of implementing some dataflows, introduces an intermediate datatype? One way to discharge this concern would be to move the writ,plea,etc definitions into a separate module IPC.Types.